.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+<!--
+
+A crisp summary
+
+Fixes #
+Ref #
+-->
+
+<!--
+Apply labels!
+-->
+
+## Checklist
+
+- [ ] Labels applied correctly? See the [contributing guidlines](../CONTRIBUTING.md).
+- [ ] Workspace compiles `cargo build --workspace`?
+- [ ] Workspace tests `cargo test --workspace`?
+- [ ] Applied `cargo +nightly fmt` with the version from CI?
+- [ ] Ran `cargo spellcheck` without any fallout? If any, update `.spellcheck/lingua.dic`.
```

Comment:
Is the full command
```
spellcheck check -cfg=.spellcheck/spellcheck.toml --checkers hunspell
```
or is list-files required first for optimization? Would be nice to include instructions how to update the dictionary exactly or a link to that.

Reply:
That's on my todo to print those instructions from `cargo-spellcheck` directly, covered by https://github.com/drahnr/cargo-spellcheck/issues/193 and planned to be done thisweek™

Reply:
Done https://github.com/paritytech/polkadot/pull/5390/commits/0d5b720886df235b28272bf962b50ea221ae15ad